modules/reactor/doc/content/source/meshgenerators/PolygonConcentricCircleMeshGenerator.md

```diff
@@ -0,0 +1,80 @@
+- `num_sectors_per_side_meta`: a `MeshMetaData` that makes the value of the input parameter `num_sectors_per_side` available to other mesh generators.
+- `azimuthal_angle_meta`: a vector `MeshMetaData` that contains sorted azimuthal angles of all the nodes on the external boundary.
+
+## Other Information
+
+By default, the mesh generated by this object contains a mixture of QUAD4 and TRI3 elements. To be specific, the central region of the mesh consists of triangular elements, whereas the rest of the mesh consists of quadrilateral elements. The users also have an option to generate a purely QUAD4 mesh by setting `quad_center_elements` as true (see [Figure 3](#TRI3_and_QUAD4)).
+
+!media media/meshgenerators/tri_quad.png
+      style=display: block;margin-left:auto;margin-right:auto;width:40%;
+      id=TRI3_and_QUAD4
+      caption=A schematic drawing showing the different meshing algorithms when `quad_center_elements` is set as `false` and `true`.
+
+When `rings` regions are present, the radii of these `rings` can be auto-adjusted to preserve the meshed annular area (volume) to the true area (volume) by setting `preserve_volumes` as true.
+
+By default, the azimuthal angles used to discretize the mesh are uniformly spaced in _angle_ on each sector of the polygon. However, uniform spacing of angles leads to non-uniform mesh intervals on the polygon external sides (unless exactly 2 sectors per side are used). However, as a uniformly spaced mesh on the polygon external boundary may facilitate mesh stitching, the user may force a uniform mesh on the polygon external boundary by setting `uniform_mesh_on_sides` as true.
+
+The default block id numbering starts with 0 at the center region and increments by 1 for each mesh region radially outward. The user may instead set specific block id and name values to rings, background, and duct regions through the use of `ring_block_ids`, `background_block_ids`, `duct_block_ids`, `ring_block_names`, `background_block_names`,  and `duct_block_names`.
+
+The user should be aware that in two cases, the central geometric region is defined as two blocks instead of one:
+- When rings are not present, and `background_intervals` > 1
+- When rings are present, and the first entry of `ring_radii` > 1
+
+In both of these cases, the central geometric region will contain both blocks 0 and 1 by default. Any additional usage of block ids or name arrays will require an extra entry for the first geometric region. This extra block definition is needed to accommodate flexibility in defining the central region as either quadrilateral or triangular elements while still stitching properly to the neighboring region.
+
+The external boundary has a default boundary id of 10000. The user may instead set specific boundary id and/or name values through the use of `external_boundary_id` and `external_boundary_name`.
```

Comment:
could we change that to a name? Like the square meshes all have default names of left right front back top bottom. 
Not sure what's a good one for an hexagon though

Reply:
This is a general polygon mesh generator so that the number of sides is not fixed. We may use `side_1`,`side_2`, etc. But it does not seem very helpful.